### DISCUSSION TITLE
Interest in continuing **alchemiscale user group** meetings?

### DISCUSSION THREAD

**Discussion by dotsdl** [OpenFreeEnergy/alchemiscale#344] [Polls]
We currently host an **alchemiscale user group** meeting every two weeks on Tuesdays at 17:00 UTC, on alternate weeks from our **alchemiscale dev group** meeting. However, attendance at the **user group** meeting has grown sparse, perhaps indicating less need for this particular forum.

Note that users are encouraged to ask questions here on GitHub Discussions, and that this forum for seeking help will continue!

**Comment by j-wags** [comment]
I voted "not needed right now" because OpenFF has all its personnel currently fully committed in other directions. At some point in the next year we'll likely have a FF to benchmark and will ask for lots of help getting started and debugging. But we're just fully committed elsewhere at the moment.

**Reply by IAlibay** [reply]
Since @dotsdl and I will likely be reviewing community needs - an important question: is OpenFF's need specifically alchemiscale or is it an openfe user need or both?

**Reply by j-wags** [reply]
It'll likely be both - In my head we want to reach a point where our workflow is "swap in a release candidate OFFXML to a standard submission script, start up some workers on NRP (or submit the job to F@H), and then have FE calcs run on the PLB". I know there are currently limitations on which system components the OFFXML can apply to, and we'll likely need to work with both the OpenFE and alchemiscale devs to resolve other issues.

**Reply by IAlibay** [reply]
I see thanks!

**Reply by dotsdl** [reply]
Thanks for this @j-wags! We can spin up regular working sessions with OpenFF as needed later this year for `alchemiscale`-specific needs as your usage increases. In the meantime, we'll work with @IAlibay on mechanisms to support users with needs for broader application science assistance.

**Comment by dotsdl** [comment]
Thanks everyone who participated in this poll! I have dropped the frequency of the **alchemiscale user group** by half. It now occurs every 4 weeks, at the same timeslot as the **alchemiscale dev group** meeting but never colliding with it.
